www/templates/user-vhost-warning

```diff
@@ -0,0 +1,8 @@
+Hi {name},
+
+We noticed that your virtual host {host.berkeley.edu} didn't have the required
+Hosted by OCF banner or University Disclaimer[2].  Please make sure you
+resolve this issue within 2 weeks time in order to avoid having it be disabled.
+
+[1] https://ocf.io/vhost#h3_including-the-ocf-banner
```

Comment:
if we're linking to this in an email, it should probably have a permanent anchor name